features/step_definitions/list-streams.rb

```diff
@@ -0,0 +1,15 @@
+Given('that I have signed-in') do
```

Comment:
```suggestion
Given('that I have signed-in') do
    visit '/'
end
```

Reply:
Shouldn't we add a pending before the `visit '/'` where we check to see if the user is signed in?